src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py

```diff
@@ -123,6 +123,18 @@ def updateAgentDrainingMode(self, agentName, drainFlag):
                                    (drainFlag, resp))
             return False
 
+    def updateSpeedDrainConfig(self, agentName, speedDrainConfig):
```

Comment:
And how about replacing these two new methods by a generic one like `updateWMAgentConfig`

Reply:
I add this as updateAgentConfig.  Given a key/value pair, update the appropriate agent aux db configuration.